solidity/contracts/token/HypERC721.sol

```diff
@@ -7,12 +7,16 @@ import {IERC721Upgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC7
 import {ERC721Upgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC721/ERC721Upgradeable.sol";
 import {ERC721EnumerableUpgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721EnumerableUpgradeable.sol";
 
+uint256 constant ERC721_SCALING_FACTOR = 1;
+
 /**
  * @title Hyperlane ERC721 Token Router that extends ERC721 with remote transfer functionality.
  * @author Abacus Works
  */
 contract HypERC721 is ERC721EnumerableUpgradeable, TokenRouter {
-    constructor(address _mailbox) TokenRouter(_mailbox) {}
+    constructor(
```

Comment:
Just wondering if you thought about having a FungibleTokenRouter that has SCALE in the constructor to avoid this. I know the inheritance tree is already super deep, but figure we talk about it

Reply:
to avoid the diff? then I would need to add more subclasses no?

Reply:
addressed